From 66d4d4ed20974e718e8a72c9c2e49f0fece61c2a Mon Sep 17 00:00:00 2001
From: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
Date: Fri, 27 Mar 2020 12:48:47 +0200
Subject: [PATCH 73/89] platform/x86: surface3_power: Prefix POLL_INTERVAL with
 SURFACE_3

For better namespace maintenance prefix POLL_INTERVAL macro with SURFACE_3.

Signed-off-by: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
---
 drivers/platform/x86/surface3_power.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/platform/x86/surface3_power.c b/drivers/platform/x86/surface3_power.c
index 4229e64a84e8..f6c646f5af93 100644
--- a/drivers/platform/x86/surface3_power.c
+++ b/drivers/platform/x86/surface3_power.c
@@ -40,8 +40,8 @@
 #include <linux/uuid.h>
 #include <asm/unaligned.h>
 
-#define POLL_INTERVAL		(2 * HZ)
-#define SURFACE_3_STRLEN 10
+#define SURFACE_3_POLL_INTERVAL		(2 * HZ)
+#define SURFACE_3_STRLEN		10
 
 struct mshw0011_data {
 	struct i2c_client	*adp1;
@@ -350,7 +350,7 @@ static int mshw0011_poll_task(void *data)
 	set_freezable();
 
 	while (!kthread_should_stop()) {
-		schedule_timeout_interruptible(POLL_INTERVAL);
+		schedule_timeout_interruptible(SURFACE_3_POLL_INTERVAL);
 		try_to_freeze();
 		ret = mshw0011_isr(data);
 		if (ret)
-- 
2.30.2

